src/shared/alertMessage.ts

```diff
@@ -7,21 +7,75 @@ aren't working. Instead, we tell them about the error
 with the hope that they either reload the page or tell
 us about the error.
 
-### Example usage
+This module provides three functions to help with this situation.
+
+### alertMessage(err)
+
+Returns a string based on err with (hopefully helpful) information about
+what is going on and what the person exploring sequences might do.
+
+#### Example usage
 
 ```typescript
 window.alert( alertMessage(someError) )
 ```
 **/
 export const alertMessage = (error: string | unknown) => {
     const errorMessage =
-        'Numberscope experienced an error.\n\n'
-        + 'Error Message:\n'
-        + `${error}\n\n`
-    const reloadDirective = 'You might need to reload the page.\n\n'
-    const sendEmailDirective =
-        'If this issue persists, please send an '
-        + 'email to numberscope@colorado.edu with steps to reproduce '
-        + 'the error and the error message.'
-    return errorMessage + reloadDirective + sendEmailDirective
+        'This visualization generated an error.\n\n' + `${error}\n\n`
+    const suggestion =
+        'Please see the "Errors" section in the User Guide, '
+        + 'accessible via the Help menu in the top bar. '
+    return errorMessage + suggestion
```

Comment:
The "Errors" section is under "Introduction" and I didn't know it was there because this was one more layer down so it isn't visible unless you expand Introduction.  Could we rephrase this?  "Please see User Guide > Introduction > Errors, accessible via the Help menu in the top bar." or something similarly detailed?